src/app/pages/LandingPage.svelte

```diff
@@ -59,6 +60,7 @@
     display: flex;
     width: 80vw;
     max-width: var(--body-max-width);
+    min-width: 600px;
```

Comment:
Ah, this helps prevent "tearing" of the background and the wave! Could you please [change it to `rem`](https://github.com/eddieantonio/predictive-text-studio/blob/production/docs/conventions.md#css-prefer-rem-to-px-for-positioning):

```suggestion
    min-width: 37.5rem;
```